elastalert/elastalert.py

```diff
@@ -1439,6 +1449,18 @@ def get_kibana_discover_external_url_formatter(self, rule):
             rule[key] = formatter
         return formatter
 
+
+    def get_opensearch_discover_external_url_formatter(self, rule):
+        """ Gets or create the external url formatter for kibana discover links """
+        key = '__kibana_discover_external_url_formatter__'
```

Comment:
Shouldn't use kibana references in an opensearch piece of logic.

Reply:
renamed it to __opensearch_discover_external_url_formatter__